package.json

```diff
@@ -34,6 +34,7 @@
     "lodash": "^4.5.1",
     "mocha": "^2.4.5",
     "mustache": "^2.2.1",
+    "npm-run-all": "^4.0.2",
```

Comment:
This addition (probably) needs to be reflected in npm-shrinkwrap.json too. See @thibaudcolas's notes at https://github.com/wagtail/wagtail/pull/3292#issuecomment-274582216